### DISCUSSION TITLE
[New Feature] C4 enhancements

### DISCUSSION THREAD

**Discussion by Crimsonum** [Vinifera-Developers/Vinifera#1484] [Feature Requests]
### Description:
In RA2 and YR, infantry commandos can destroy vehicles with C4. In addition, they can place C4 charges on Bridge Repair Huts, which causes the whole bridge to detonate. Both are useful tactics and open up more possibilities in mission design.

### Possible Implementation:
General bool keys `AllowC4OnBridges=` and `AllowC4OnVehicles=`. If set, allows any unit with C4=yes to target and destroy bridges (via bridge repair huts) and vehicles, respectively.

**Comment by evanb90** [comment]
Just to clarify, Tanya in YR attacks vehicles using her Secondary= weapon, as her primary is incapable of damaging vehicle armor types. SabotageCursor=yes gives it the C4 cursor.

**Comment by Crimsonum** [comment]
Thanks for the clarification. I guess the same could be done here vehicle targeting, if custom cursors are to be supported.
